libcontainer/configs/cgroup_unix.go

```diff
@@ -13,13 +13,18 @@ const (
 type Cgroup struct {
 	Name string `json:"name"`
```

Comment:
Can you add a comment saying these fields are deprecated and omitempty to `Name` and `Parent` ?